README.md

```diff
@@ -1,41 +1,37 @@
-<p align="center">
-  <a href="http://nestjs.com/" target="blank"><img src="https://t3.ftcdn.net/jpg/02/05/78/12/360_F_205781253_acxA4jXNLyZN3XLFb7h3ySrXAlksPvXq.jpg" /></a>
-</p>
 
-## Rings Quest Backend
+## Rings Quest Ops
 
 ### Run all services with docker compose
 
 #### Prerequisites
-
-Before you run on docker, make sure that you have the following requirements installed on your system:
+Before you begin, make sure that you have the following requirements installed on your system:
+- [Node.js](https://nodejs.org/) (version 12 or higher)
+- [npm](https://www.npmjs.com/) (version v18.12.1)
 - [Docker](https://docs.docker.com/get-docker/)
-- [Docker Compose](https://docs.docker.com/compose/install/)
 
-#### Pull and run from dockerhub
+
+Make sure you have pulled the latest git update by writing in the terminal
+```shell
+ $ git pull 
+```
+#### run the following commands in the terminal
 ```shell
 $ docker-compose pull
+$ docker-compose build
 $ docker-compose up
 ````
+#### in case of failing in the last commands (may happen in the first run), try to run:
 
-On the first time running this, the app container should be failed:
-1. Go to the DB section below and follow the orders there
```

Comment:
Do not remove this instruction - it's essential because only the first time, the DB creation part is necessary before this part.